src/javascript/app/pages/trade/markets.jsx

```diff
@@ -68,11 +68,19 @@ const List = ({
 class Markets extends React.Component {
     constructor (props) {
         super(props);
-        let market_symbol = Defaults.get('market');
+        let market_symbol,underlying_symbol;
+        const login_id = localStorage.getItem('active_loginid');
```

Comment:
we usually get these values from Client, not from localstorage. please check the rest of the code on how loginid is usually retrieved.

Reply:
checked with Client, but after logout selected market and underlying cleared, so after relogin cannot and retrieve these values by Client.